src/js/utils/media.js

```diff
@@ -57,3 +58,16 @@ export function resizeImage(file) {
         return canvas.toDataURL('image/jpeg');
     });
 }
+
+export function mergeMessages(response, storeMessages, message) {
+    const responseMessages = response.conversation.messages;
+    const messages = _.filter(
+        _.unique(
```

Comment:
did you experience a case where you had double messages? the server should return message in double